pkg/portal/portal.go

```diff
@@ -261,21 +265,41 @@ func (p *portal) unauthenticatedRoutes(r *mux.Router) {
 }
 
 func (p *portal) aadAuthenticatedRoutes(r *mux.Router) {
-	for _, name := range AssetNames() {
-		if name == "v1/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/").HandlerFunc(p.index)
-			continue
-		}
+	var names []string
 
-		if name == "v2/build/index.html" {
-			r.NewRoute().Methods(http.MethodGet).Path("/v2").HandlerFunc(p.indexV2)
-			continue
+	err := filepath.Walk(".", func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			log.Fatal(err)
+			return nil
 		}
 
-		fmtName := strings.TrimPrefix(name, "v1/build/")
-		fmtName = strings.TrimPrefix(fmtName, "v2/build/")
+		names = append(names, path)
+		return nil
+	})
+
```

Comment:
```suggestion
```
Makes it clearer that this error check is related to the above Walk

Reply:
not sure I understand your suggestion here @cblecker